keyboards/hub16/matrix.c

```diff
@@ -222,8 +222,7 @@ uint8_t matrix_scan(void) {
 
     debounce(raw_matrix, matrix, MATRIX_ROWS, changed);
 
-    // Read encoder switches, already debounced
-    changed |= read_encoder_values(matrix, 4);    
+    changed |= read_encoder_switches(matrix, 4);
```

Comment:
As you have removed the debounce logic, wouldnt the reading need to be moved before the `debounce()` call?

Reply:
I thought the exact same thing, however if I place it before the `debounce()` call it will send the key multiple times rapidly even if tapped once. Hadn't had any issues with the above and couldn't figure out a better solution so pushed it.

Looked back at the code now and found the issue in my debounce routine, thanks for the poke to do this properly.